apps/src/code-studio/showProjectAdmin.js

```diff
@@ -23,6 +23,47 @@ export default () => {
     }
   }
 
+  if ($('#feature_project').length) {
+    if (dashboard.project.isProjectLevel()) {
+      var featuredProjectId = $('.featured_project_id').data('id');
+      var deleteUrl = '/featured_projects/' + featuredProjectId  + '/' + dashboard.project.getCurrentId();
```

Comment:
if you take my comment in routes.rb, then you could remove `featuredProjectId  + '/'` from this URL here.

Reply:
nit: prefer string interpolation